### PR TITLE
Mahabharat file added under Hindi sub-directory.

### PR DESCRIPTION
**Pull request by surajkumarcse**
Please review this, @casey-collab

### CODE REVIEW

File: hindi/Mahabharat/mahabharat.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Indian culture is filled with the learnings of [Mahabharata](https://www.hotstar.com/tv/mahabharat/s-28)
```
Comment (casey-collab) [inline hindi/Mahabharat/mahabharat.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by surajkumarcse** [comment]
Please review this, @casey-collab

File: hindi/Mahabharat/mahabharat.md (hunk 1 above)
Comment (casey-collab) [inline hindi/Mahabharat/mahabharat.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by surajkumarcse** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :fireworks:

**Pull request merged**